.gitignore

```diff
@@ -24,23 +24,16 @@ project.xcworkspace
 
 # Android/IJ
 #
+*.iml
 .idea
 .gradle
 local.properties
-*.iml
 
 # node.js
 #
 node_modules/
-coverage/
 npm-debug.log
 
-# Secrets
-env.js
```

Comment:
I think it should. Also this PR contains a `env.js` file that should be removed as well 🙂